src/Ocean/HydrostaticBoussinesq/HydrostaticBoussinesqModel.jl

```diff
@@ -671,7 +671,7 @@ function update_auxiliary_state_gradient!(
 
         return nothing
     end
-    nodal_update_auxiliary_state!(f!, dg, m, Q, t, elems; diffusive = true)
+    update_auxiliary_state!(f!, dg, m, Q, t, elems; diffusive = true)
```

Comment:
If I renamed `f!` to `nodal_update_auxiliary_state!`, could I use the new function signature you added? Also, could I move the definition of `f!` outside of `update_auxiliary_state_gradient!` and still have it work?

Reply:
> If I renamed f! to nodal_update_auxiliary_state!, could I use the new function signature you added

Good point. Not at the moment, but we should be able to do that if we add/propagate that kwarg. I'll update.

> Also, could I move the definition of `f!` outside of update_auxiliary_state_gradient! and still have it work?
Yes, I believe so if we change `f!` to `nodal_update_auxiliary_state!`.

Reply:
Alright, done.